libs/tokens/src/utils/tokenMapToList.ts

```diff
@@ -0,0 +1,10 @@
+import { TokenWithLogo } from '@cowprotocol/common-const'
+
+import { TokensMap } from '../types'
+import { tokenWithLogoFromToken } from './tokenWithLogoFromToken'
+
+export function tokenMapToList(tokenMap: TokensMap): TokenWithLogo[] {
```

Comment:
should indicate that this will sort alphabetically in the JSdocs

Reply:
Added: https://github.com/cowprotocol/cowswap/pull/3201/commits/20af0e7a3782ed1c30b680532a3f68168375b495